src/js/app.jsx

```diff
@@ -1,7 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { library } from "@fortawesome/fontawesome-svg-core";
+import {
+    faAngleDown,
+    faSpinner,
+    faTimes
+} from "@fortawesome/free-solid-svg-icons";
```

Comment:
It's to prevent the line from getting too long as we add more icons. See https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/app.jsx#L6 for example